NEWS.rst

```diff
@@ -2,10 +2,39 @@
 NEWS
 ******
 
+Version 0.13
```

Comment:
Wait, do we want to write the 0.13 already?

Reply:
That is the next version we are releasing, and there are commits that were not backported to `v0.12` that have been merged since the specified commit b9f3036484d850401fd6d7850b6eeba6d0bd0fca . Is there an alternative method I should be following? If so, we should have it documented.